{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "c2761979_402882b1",
        "filename": "plugins/tripleo-undercloud/tasks/upgrade/generate_files.yml",
        "patchSetId": 4
      },
      "lineNbr": 19,
      "author": {
        "id": 1017908
      },
      "writtenOn": "2022-11-08T13:57:28Z",
      "side": 1,
      "message": "Thanks a lot Lukas for the fix, yes for sure we need to be root.",
      "revId": "6ea236cddc6ecc112543c64d3b4c75167f915e2e",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    }
  ]
}